watertap/core/zero_order_costing.py

```diff
@@ -661,6 +663,58 @@ def cost_deep_well_injection(blk):
         blk.config.flowsheet_costing_block.cost_flow(
             blk.unit_model.electricity[t0], "electricity")
 
+    def cost_dmbr(blk):
+        """
+        General method for costing dynamic membrane bioreactor. Capital cost
+        is based on the cost of membrane area.
+
+        This method also registers the electricity demand as a costed flow.
+        """
+        t0 = blk.flowsheet().time.first()
+
+        # Get parameter dict from database
+        parameter_dict = \
+            blk.unit_model.config.database.get_unit_operation_parameters(
+                blk.unit_model._tech_type,
+                subtype=blk.unit_model.config.process_subtype)
+
+        # Get costing parameter sub-block for this technology
+        A, B = _get_tech_parameters(
+            blk,
+            parameter_dict,
+            blk.unit_model.config.process_subtype,
+            ["water_flux",
+             "reactor_cost"])
+
+        # Add cost variable and constraint
+        blk.capital_cost = pyo.Var(
+            initialize=1,
+            units=blk.config.flowsheet_costing_block.base_currency,
+            bounds=(0, None),
+            doc="Capital cost of unit operation")
+
+        Q_treated = pyo.units.convert(blk.unit_model.properties_treated[t0].flow_vol,
+                              to_units=pyo.units.L / pyo.units.hour)
+
+        ES_reactor = (1/pyo.units.convert(A, to_units=pyo.units.L / pyo.units.m**2 / pyo.units.hour))*Q_treated
+
+        expr = pyo.units.convert(ES_reactor*B, to_units=blk.config.flowsheet_costing_block.base_currency)
```

Comment:
You could save another unit conversion here by combining this with the one above.

```
expr = pyo.units.convert(
    blk.unit_model.properties_treated[t0].flow_vol/A*B,
    to_units=blk.config.flowsheet_costing_block.base_currency)
```